.github/workflows/publish.yml

```diff
@@ -25,6 +25,9 @@ jobs:
         run: |
           git config --global user.email ${{ github.actor }}@users.noreply.github.com
           git config --global user.name ${{ github.actor }}
+      
+      - name: Run cdnCheck script
```

Comment:
I'm wondering if this should be changed to something like 'Ensure SVGs have been published on CDN' so that readers can get context about what the script is doing without reading the source.